app/pandas.ipynb

```diff
@@ -333,114 +908,302 @@
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use ** operator here"
+    "power_s = s1.pow(s2)\n",
+    "power_s"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-08T12:01:21.034661Z",
+     "start_time": "2024-11-08T12:01:21.029936Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0          1\n",
+       "1        128\n",
+       "2       6561\n",
+       "3     262144\n",
+       "4    9765625\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 28
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use // operator here"
+    "result_s = s2.floordiv(s1)\n",
+    "result_s"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-08T12:02:13.374073Z",
+     "start_time": "2024-11-08T12:02:13.369525Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    6\n",
+       "1    3\n",
+       "2    2\n",
+       "3    2\n",
+       "4    2\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 29
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use % operator here"
+    "mod_s = s2.mod(s1)\n",
+    "mod_s"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-08T12:02:46.471762Z",
+     "start_time": "2024-11-08T12:02:46.466927Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    0\n",
+       "1    1\n",
+       "2    2\n",
+       "3    1\n",
+       "4    0\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 30
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s1 here"
+    "filtered_s = s1[s1 % 2 == 0]\n",
+    "filtered_s"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-08T12:03:58.627800Z",
+     "start_time": "2024-11-08T12:03:58.623086Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1    2\n",
+       "3    4\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 31
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s2 here"
+    "filtered_s2 = s2[s2 % 2 == 0]\n",
```

Comment:
The task requires filtering `s2` for odd numbers, but your current implementation filters for even numbers. Change the condition to `s2 % 2 != 0` to correctly filter for odd numbers.